database/images/packer/build.sh

```
@@ -0,0 +1,148 @@
+  sudo apt-get install -y -qq qemu-system-x86 qemu-utils ovmf
+fi
+
+cleanup() {
+  rm -rf "$KEY_DIR"
+}
+trap cleanup EXIT
+
+ssh-keygen -t ed25519 -f "$KEY_FILE" -N "" -C "packer-build" -q
+
+cp "$META_DATA_TEMPLATE" "$SEED_DIR/meta-data"
+sed "s|PACKER_SSH_PUBLIC_KEY_PLACEHOLDER|$(cat ${KEY_FILE}.pub)|g" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)